### ISSUE TITLE
Option to disable screenshots: @Screenshots(disabled=true)

### ISSUE THREAD

**Issue by splintrx** [serenity-bdd/serenity-core#188]
From this topic:
https://groups.google.com/forum/m/#!msg/thucydides-users/l1MjDbZzjFk/m02Yb2ZcKgAJ

"We could probably improve this to something like @Screenshots(disabled=true) if there was interest."
+1 for this feature

We have some non-web tests (REST tests) in the suite.

**Comment by YamStranger** [comment]
Hi @splintrx!

Thanks for reporting us. Screenshots annotation updated and available under [v1.1.22-rc.6](https://github.com/serenity-bdd/serenity-core/releases/tag/v1.1.22-rc.6). You can check it using our demo project 
[junit-webtest](https://github.com/serenity-bdd/serenity-demos/tree/master/junit-webtests) with last 
Here is example for default behaving:
```
@RunWith(SerenityRunner.class)
public class SearchByKeywordTest {
    @Managed
    WebDriver driver;

    @Steps
    BuyerSteps buyer;

    @Test
    @WithTags( {
            @WithTag(type="priority",name="medium"),
            @WithTag(type="component",name="search")

    })
    public void search_for_articles_by_keyword() {
        buyer.opens_home_page();
        buyer.searches_by_keyword("wool");
        buyer.should_see_results_summary_containing("wool");
    }

    @Test//(timeout = 100)
    public void search_for_articles_by_shop_name() {
        buyer.opens_home_page();
        buyer.searches_for_shop_called("docksmith");
        buyer.should_see_shop_search_result_summary_of("1 shop found for docksmith");
    }
}
```
report will contains all screenshots:
![report_with_images](https://cloud.githubusercontent.com/assets/1667699/11265681/04a47376-8ea7-11e5-97a2-af1ff8275c51.png)


To disable screenshots just add @Screenshots(disabled=true):
```
RunWith(SerenityRunner.class)
public class SearchByKeywordTest {
    @Managed
    WebDriver driver;

    @Steps
    BuyerSteps buyer;

    @Test
    @WithTags( {
            @WithTag(type="priority",name="medium"),
            @WithTag(type="component",name="search")

    })
    @Screenshots(disabled=true)
    public void search_for_articles_by_keyword() {
        buyer.opens_home_page();
        buyer.searches_by_keyword("wool");
        buyer.should_see_results_summary_containing("wool");
    }

    @Test//(timeout = 100)
    @Screenshots(disabled=true)
    public void search_for_articles_by_shop_name() {
        buyer.opens_home_page();
        buyer.searches_for_shop_called("docksmith");
        buyer.should_see_shop_search_result_summary_of("1 shop found for docksmith");
    }
}
```
In this case report will be without screenshots:

![report_without_images](https://cloud.githubusercontent.com/assets/1667699/11265719/55aa4ebc-8ea7-11e5-9e44-04a91a5c49e2.png)

If you haven't additional questions please close this issue. 

Thanks!!!

**Comment by mohamedb** [comment]
I still have all the pictures event when I use the annotation @Screenshots(disabled=true)
